.github/workflows/stalled.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           stale-pr-label: 'stalled'
           stale-pr-message: 'This PR is stalled because it has been open for 30 days with no activity. Remove stalled label or comment or this will be closed in 7 days.'
           close-pr-message: 'This PR was closed because it has been stalled for 7 days with no activity.'
```

Comment:
nit: Do we need to remove numbers 7, 30 or change to X until we decide on right number

Reply:
Deleted the file all together!